applications/altran-v2x/readme.md

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Altran Technology
+
+Overview of Altran V2X Apps:
+=============================
+
+Altran V2X solution enables safety & non-safety applications for V2V,V2I,V2P and V2N scenarios for Road Side Units (RSU), On Board Units (OBU) and Smart Pedestrian Devices . The solution comes with application and underlying stack conformant to regional standards - WAVE / ETSI and China.
+
+V2X Application Description:    
+==============================
+Following Vehicle To Infrastructure (V2I) applications are supported:
+
+Roadside Alert (RSA): RSU periodically broadcast advisory messages about the road hazards (Slippery Road, Animal On the Road etc.)
+
+Electric Vehicle Charging Station (EVCSN): RSU periodically broadcast presence and occupancy status of nearby Electric changing station Information
+
+Vulnerable Road User (VRU): RSU detect vulnerable road user and broadcast the same information to near-by vehicles
+
+Time To Green (TTG) / Green Light Signal (GLOSA): RSU broadcast the traffic signals state and time information to near-by vehicles in an intersection
+
+Separate container images are provided for OBU / RSU apps which can be onboarded (as pods) and communicate over UDP/IP interface. A Human Machine Interface (HMI) app is provided for visualization and demo purpose. In real deployment scenario the UDP/IP interface could be replaced by C-V2X PC5 or C-V2X Uu air interface
+
+Retrieve Container Images and Helm Charts
+
+Containers: Contact Altran Sales
+
+Helm Charts: https://gurftp03.aricent.com/webclient/Login.xhtml
+
+Pre-Requisite:
```

Comment:
Which deployment flavor of OpenNESS experience kit is used for this Application Demonstration?

Reply:
Please provide the details of the OpenNESS version that is used for verifying this V2X application to ensure the users are having the details of the OpenNESS.

Reply:
These comments are not yet resolved?